src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -45,17 +48,12 @@ type PolicyOption = ListItem & {
     keyForList: string;
 };
 
-type WorkspaceCategoriesOnyxProps = {
+type WorkspaceCategoriesPageProps = {
     /** The policy the user is accessing. */
     policy: OnyxEntry<OnyxTypes.Policy>;
-
-    /** Collection of categories attached to a policy */
-    policyCategories: OnyxEntry<OnyxTypes.PolicyCategories>;
 };
 
-type WorkspaceCategoriesPageProps = WorkspaceCategoriesOnyxProps & StackScreenProps<WorkspacesCentralPaneNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORIES>;
-
-function WorkspaceCategoriesPage({policy, policyCategories, route}: WorkspaceCategoriesPageProps) {
+function WorkspaceCategoriesPage({policy}: WorkspaceCategoriesPageProps) {
```

Comment:
Revert. We should use `route` from the props. Will look into the types errors...

Reply:
Export `WithPolicyConnectionsProps` and use it
```js
type WorkspaceCategoriesPageProps = WithPolicyConnectionsProps & WorkspaceCategoriesPageOnyxProps;
```

Reply:
thank you very much for figuring out the type issues 😄